core/src/main/kotlin/com/google/container/tools/core/util/UsageTrackerBundle.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.core.util
+
+import com.intellij.CommonBundle
+import org.jetbrains.annotations.PropertyKey
+import java.util.ResourceBundle
+
+private const val BUNDLE_NAME = "messages.UsageTrackerBundle"
+
+/**
+ * Message bundle manager for usage tracking related featured.
+ */
+object UsageTrackerBundle {
```

Comment:
Could we possibly merge the two bundles (core and usage tracker) into one bundle, probably just `core`? It feels very fine grained and mildly confusing since the core has only one string and probably the usage tracker will stay in the core module anyway? I believe we usually chose to have a single bundle per module in our cloud tools plugin.